recipes/orthofisher/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "orthofisher" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/orthofisher-{{ version }}.tar.gz
+  sha256: e8600178d630f1978cf8c6657a6ff128c309dc6bc3bb6f6549ffc04ed4e02e29
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
+  entry_points:
+    - orthofisher = orthofisher.orthofisher:main
+  run_exports:
+    - {{ pin_subpackage('orthofisher', max_pin="x") }}
```

Comment:
`max_pin="x.x"` ? since the version = "x.x.x"

Reply:
I believe the subpackage should be `max_pin="x"`, since the package follows semantic versioning (x.x.x) and does not have known CLI breakages in minor versions, no?